src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball1 = new Lottery().getRandomBall();
+        Ball ball2 = new Lottery().getRandomBall();
+        Ball ball3 = new Lottery().getRandomBall();
```

Comment:
You are creating multiple instances of the `Lottery` class. According to the checklist, it's better to create a single instance and reuse it. Consider creating one `Lottery` instance and using it to generate all three balls.